zebra-consensus/src/block.rs

```diff
@@ -21,19 +21,15 @@ use thiserror::Error;
 use tower::{Service, ServiceExt};
 use tracing::Instrument;
 
-use zebra_chain::{
-    amount::Amount,
-    block::{self, Block},
-    parameters::Network,
-    transparent,
-    work::equihash,
-};
+use zebra_chain::{amount::Amount, block, parameters::Network, transparent, work::equihash};
 use zebra_state as zs;
 
 use crate::{error::*, transaction as tx, BoxError};
 
 pub mod check;
+pub mod request;
 pub mod subsidy;
+pub use request::Request;
```

Comment:
Nitpick: I usually separate different types of imports from each other for readability.

```suggestion

pub use request::Request;
```